markdown.go

```diff
@@ -99,8 +99,17 @@ Flags are added through the query string. For example, the %s flag is the %s que
 }
 `+"```"+`
 
+Some flags may be repeated. For example, the --status flag may be reused as below:
+
+`+"```"+`
+> curl -X POST "http://127.0.0.1:5001/api/v0/pin/remote/service/ls?name=myservice&status=pinned&status=pinning"
+`+"```"+`
+
 ::: tip
 Some arguments may belong only to the CLI but appear here too. These usually belong to client-side processing of input, particularly in the `+"`add`"+` command.
+
+Additionally, as a convenince certain CLI commands may allow passing repeated flags as delimited lists such as
```

Comment:
Slight clarification and typo fix here:

```suggestion
Additionally, as a convenience certain CLI commands may allow passing repeated flags as comma-delimited lists such as
```

Reply:
The fact that the list is comma delimited is technically per-CLI command. i.e. if some command took lists of comma separated tuples like `cmd --tuples=(a,b) --tuples=(c,d)` the cmd command might decide on a different delimiter like `;` and so we'd be able to get `cmd --tuples=(a,b);(c,d)`.

Given that the only commands with delimiters use commas, I'd be ok with this change although we run the risk of accidentally not updating this if we add commands with different delimiters.

Reply:
Good point! I still think the language could be clarified a bit (so someone doesn’t assume an _implied_ “comma” there). Maybe:

> some CLI commands allow passing repeated flags as delimited lists (delimiters vary by command). For example: …